source/_integrations/philips_js.markdown

```diff
@@ -17,12 +19,15 @@ ha_platforms:
   - media_player
   - remote
   - switch
+  - sensor
+  - binary_sensor
```

Comment:
PRs to the Core codebase may not contain multiple platforms in a single PR, therefore, this PR can also not contain multiple platforms being added in a single PR.